### PR TITLE
Rchriste/n5010 test esram

### PR DESCRIPTION
**Pull request by rchriste1**

### CODE REVIEW

**Review by anandaravuri — DISMISSED** [review]

File: samples/n5010-test/n5010-test.c [hunk 1]
```diff
@@ -443,6 +473,138 @@ static fpga_result fpga_test_ddr_directed(struct n5010 *n5010)
...
+			printf("timeout (16x eSRAM channels): 0x%04jx\n", timeout);
+		}
+		nanosleep(&ts, NULL);
+
+	} while ( (pass | fail | timeout) != 0xffff);
+
+	if (fail != 0x0) {
+		fprintf(stderr, "Error: Test failed on the following channels: 0x%04jx\n", fail);
+		res = FPGA_EXCEPTION;
+	};
```
Comment (tswhison) [inline samples/n5010-test/n5010-test.c:523]: Please remove unneeded ;

File: samples/n5010-test/n5010-test.c [hunk 2]
```diff
@@ -443,6 +473,138 @@ static fpga_result fpga_test_ddr_directed(struct n5010 *n5010)
...
+
+	if (timeout != 0x0) {
+		fprintf(stderr, "Error: Test timed out on the following channels: 0x%04jx\n", timeout);
+		res = FPGA_EXCEPTION;
+	};
```
Comment (tswhison) [inline samples/n5010-test/n5010-test.c:528]: Please remove unneeded ;

File: samples/n5010-test/n5010-test.c [hunk 3]
```diff
@@ -443,6 +473,138 @@ static fpga_result fpga_test_ddr_directed(struct n5010 *n5010)
...
+
+	if (pass != 0xffff) {
+		fprintf(stderr, "Error: Test did not pass on all channels: 0x%04jx\n", pass);
+		res = FPGA_EXCEPTION;
+	};
```
Comment (tswhison) [inline samples/n5010-test/n5010-test.c:533]: Please remove unneeded ;

File: samples/n5010-test/n5010-test.c [hunk 4]
```diff
@@ -443,6 +473,138 @@ static fpga_result fpga_test_ddr_directed(struct n5010 *n5010)
...
+
+	if (fail != 0x0) {
+		fprintf(stderr, "Error: Test failed on the following channels: 0x%02jx\n", fail);
+		res = FPGA_EXCEPTION;
+	};
+
+	if (timeout != 0x0) {
+		fprintf(stderr, "Error: Test timed out on the following channels: 0x%02jx\n", timeout);
+		res = FPGA_EXCEPTION;
+	};
```
Comment (tswhison) [inline samples/n5010-test/n5010-test.c:594]: Please remove unneeded ;

File: samples/n5010-test/n5010-test.c [hunk 5]
```diff
@@ -443,6 +473,138 @@ static fpga_result fpga_test_ddr_directed(struct n5010 *n5010)
...
+
+	if (pass != 0xff) {
+		fprintf(stderr, "Error: Test did not pass on all channels: 0x%02jx\n", pass);
+		res = FPGA_EXCEPTION;
+	};
```
Comment (tswhison) [inline samples/n5010-test/n5010-test.c:599]: Please remove unneeded ;

File: samples/n5010-test/n5010-test.c [hunk 6]
```diff
@@ -443,6 +473,138 @@ static fpga_result fpga_test_ddr_directed(struct n5010 *n5010)
...
+			printf("timeout (8x QDR channels): 0x%02jx\n", timeout);
+		}
+		nanosleep(&ts, NULL);
+
+	} while ( (pass | fail | timeout) != 0xffff);
+
+	if (fail != 0x0) {
+		fprintf(stderr, "Error: Test failed on the following channels: 0x%02jx\n", fail);
+		res = FPGA_EXCEPTION;
+	};
```
Comment (tswhison) [inline samples/n5010-test/n5010-test.c:589]: Please remove unneeded ;

**Review by tswhison — APPROVED** [review]

**Pull request merged**